.upstream-tests/test/std/atomics/atomics.types.generic/atomic_copyable.pass.cpp

```diff
@@ -44,21 +45,23 @@ void test_copy_constructible() {
 template <class T, class A>
 __host__ __device__
 void test_atomic_ref_copy_ctor() {
-  A val = 0;
+  SHARED A val;
+  val = 0;
 
   T t0(val);
   T t1(t0);
 
-  t0++;
-  t1++;
+  // t0++;
+  // t1++;
 
-  assert(t1.load() == 2);
+  // assert(t1.load() == 2);
```

Comment:
I assume that these weren't meant to be left commented out?

Reply:
Fixed! Thanks.